addons/binding/org.openhab.binding.bigassfan/src/main/java/org/openhab/binding/bigassfan/discovery/BigAssFanDiscoveryService.java

```diff
@@ -254,7 +244,7 @@ public void run() {
                 try {
                     discoveryListener.pollForDevices();
                 } catch (Exception e) {
-                    logger.debug("Poll job got unexpected exception: {}", e.getMessage());
+                    logger.error("Poll job got unexpected exception: {}", e.getMessage(), e);
```

Comment:
Can you make this a warning errors are only for things threatening the stability of the complete openHAB instance.